.appveyor.yml

```diff
@@ -35,11 +35,12 @@ install:
 
 before_test:
     - cd C:\projects\php-cs-fixer
-    - php C:\tools\composer.phar update --optimize-autoloader --no-interaction --no-progress --prefer-stable --no-ansi
-    - php C:\tools\composer.phar info -D | sort
+    - php C:\tools\composer.phar remove --dev --no-update --working-dir=./dev-tools humbug/box localheinz/composer-normalize phpstan/phpstan-phpunit
```

Comment:
why `remove` ?

Reply:
One job runs on PHP 5.6 and these packages need 7+.

Reply:
yet, on stable branch, there is no issue like that in AppVeyor
https://ci.appveyor.com/project/keradus/php-cs-fixer/builds/29070411

Reply:
Because those packages are already in dev-tools.